tests/integration/remote/invoke/remote_invoke_integ_base.py

```diff
@@ -58,6 +58,8 @@ def create_resources_and_boto_clients(cls):
         }
         cls.cfn_client = get_boto_client_provider_with_config()("cloudformation")
         cls.lambda_client = get_boto_client_provider_with_config()("lambda")
+        cls.stepfunctions_client = get_boto_client_provider_with_config()("stepfunctions")
+        cls.xray_client = get_boto_client_provider_with_config()("xray")
```

Comment:
why do we need to call `get_boto_client_provider_with_config()` 4 times?

Reply:
Good callout, can create a variable and create a single call.